src/controller/CommissioningDelegate.h

```diff
@@ -553,6 +555,16 @@ class CommissioningParameters
         return *this;
     }
 
+    // Check for Joint Commissioning Method
+    Optional<bool> UseJCM() const { return mUseJCM; }
```

Comment:
Should be renamed to GetUsedJCM, all the other getters from this file seems to be prefixed by "Get" followed by the variable name.